### ISSUE TITLE
Delete some labels

### ISSUE THREAD

**Issue by gvanrossum** [python/mypy#2645]
Not all labels are equally useful. I am going to delete the following labels (the stats were accurate as of about a month ago but still indicative of the use each label has seen):

- duplicate (48 closed); doesn’t feel useful; just add “duplicate of XXX” in the comment
- needs repro (2 closed) — we clearly don’t use this; just ask for a repro if needed
- wontfix (7 closed) — a comment with an explanation seems friendlier
- postponed (3 open, 2 closed) — subsumed into "priority-low", or just close as won’t fix
- task (8 open, 35 closed) — seems questionable, we have enough other categories
- Bytes-Unicode (rarely used, not effective)
- affects/stubgen (used only once, ever)
- difficulty/easy (not effective, may attract labor-intensive one-off contributors)
- Bytes-Unicode (4 open, 1 closed) -- not an area of much activity
- stubs (46 closed) — currently also used for issues that are really typeshed issues, but those I propose to just close with a request to open a typeshed issue instead

**Comment by ambv** [comment]
Note: this (and Milestones) requires changes to CONTRIBUTING.md, I'll add a pull request for that in a sec.

BTW, I'm replicating your change in python/typeshed now.
